tests/test_create.py

```diff
@@ -56,41 +65,47 @@ def install_in_env(env_path, *packages):
     sub_parsers = p.add_subparsers(metavar='command', dest='cmd')
     install_configure_parser(sub_parsers)
 
-    command = "install -y -p {0} {1}".format(env_path, " ".join(packages))
+    command = "install -y -q -p {0} {1}".format(env_path, " ".join(packages))
 
     args = p.parse_args(split(command))
     args.func(args, p)
 
 
 def test_just_python3():
-    with make_temp_env("python=3") as env_path:
-        assert exists(join(env_path, bindir, 'python3'))
+    with disable_dotlog():
+        with make_temp_env("python=3") as env_path:
+            assert exists(join(env_path, bindir, 'python3'))
 
 
 def test_just_python2():
-    with make_temp_env("python=2") as env_path:
-        assert exists(join(env_path, bindir, 'python2'))
+    with disable_dotlog():
+        with make_temp_env("python=2") as env_path:
+            assert exists(join(env_path, bindir, 'python2'))
 
 
 def test_python2_install_numba():
-    with make_temp_env("python=2") as env_path:
-        assert exists(join(env_path, bindir, 'python2'))
-        install_in_env(env_path, "numba")
-        assert isfile(join(env_path, bindir, 'numba'))
+    with disable_dotlog():
+        with make_temp_env("python=2") as env_path:
+            assert exists(join(env_path, bindir, 'python2'))
+            install_in_env(env_path, "numba")
+            assert isfile(join(env_path, bindir, 'numba'))
 
 
+@pytest.mark.timeout(600)
 def test_dash_c_usage_replacing_python():
-    with make_temp_env("-c conda-forge python=3.5") as env_path:
-        assert exists(join(env_path, bindir, 'python3.5'))
-        install_in_env(env_path, "decorator")
-        # TODO @mcg1969: now what?
+    with disable_dotlog():
+        with make_temp_env("-c conda-forge python=3.5") as env_path:
+            assert exists(join(env_path, bindir, 'python3.5'))
+            install_in_env(env_path, "decorator")
+            # TODO @mcg1969: now what?
```

Comment:
Maybe `any(p.startswith('conda-forge::python-3.5' for p in conda.install.linked()))`